src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -152,7 +152,21 @@ public TestCase updateTestCase(
     if (measure.getTestCases() == null) {
       measure.setTestCases(new ArrayList<>());
     }
-
+    if (!username.equalsIgnoreCase(measure.getCreatedBy())
```

Comment:
This changed after the introduction of MeasureSet(has `owner` and `acls`). We should use MeasureSet to check the permissions instead of `measure.createdBy` and `measure.acls`

I would suggest to use `measureService.findMeasureById(id)` to get the measure. This will give you measure with masureSet. Then call `measureService.verifyAuthorization()`